src/app/shared/mock-data.service.ts

```diff
@@ -186,6 +192,15 @@ export class MockDataService {
     return this.makeCopy(this.workItemLinks);
   }
 
+  public getWorkItemLinksForWorkItem(workItemId: string): any {
+    var result: any = [];
+    for (var i = 0; i < this.workItemLinks.length; i++)
+      if (this.workItemLinks[i].relationships.source.data.id === workItemId) {
```

Comment:
Not necessarily current work item has to be in source, it can be in destination as well.